tests/test_interface.rb

```diff
@@ -40,7 +40,8 @@ class TestInterface < CiscoTestCase
   DEFAULT_IF_IP_PROXY_ARP = false
   DEFAULT_IF_IP_REDIRECTS = true
   DEFAULT_IF_VRF = ''
-  IF_DESCRIPTION_SIZE = 243 # SIZE = VSH Max 255 - "description " keyword
+  IF_DESCRIPTION_VALID_SIZE = 68 # SIZE = VSH Max 80 - "description " keyword
+  IF_DESCRIPTION_INVALID_SIZE = 255
```

Comment:
@mikewiebe @glennmatthews 

The 5k and 6k platforms failed the `test_interface_description_valid` test with the following error:

```
  1) Error:
TestInterface#test_interface_description_valid:
Cisco::CliError: CliError: ' description bcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuv' rejected with message:
'ERROR: error: name too long. max size allowed is 80'
```

I made `VALID_SIZE` small enough that the valid test would pass on all platforms, and I made `INVALID_SIZE` large enough that it would raise an error for all platforms.  I'm not sure what you mean by "what falls in the middle?"